packages/mobile/src/invite/EnterInviteCode.tsx

```diff
@@ -67,6 +67,7 @@ export class EnterInviteCode extends React.Component<Props, State> {
         // Note: redux state reset is handled by UseBackToWelcomeScreen
         // tslint:disable-next-line: jsx-no-lambda
         onPress={() => navigate(Screens.Welcome)}
+        eventName={OnboardingEvents.invite_redeem_cancel}
```

Comment:
You can also cancel by hitting the hardware back button on Android or doing a swipe back on iOS.

This event should be moved to `<UseBackToWelcomeScreen ... />` which handles all of these cases, though when you've already started the redeem process we should send both `OnboardingEvents.invite_redeem_cancel` and `OnboardingEvents.create_account_cancel`. 
`UseBackToWelcomeScreen` can currently only send one event so it would need to be modified to be able to send more than one.

Reply:
Didn't realize we had that. Thanks, will use it instead!